hasher-matcher-actioner/webapp/src/pages/MatchIndex.jsx

```diff
@@ -0,0 +1,251 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+import React, {useState, useEffect} from 'react';
+import {useHistory} from 'react-router-dom';
+import {PropTypes} from 'prop-types';
+import {
+  Container,
+  Col,
+  Button,
+  Row,
+  Collapse,
+  InputGroup,
+  Form,
+  FormControl,
+} from 'react-bootstrap';
+import Spinner from 'react-bootstrap/Spinner';
+import classNames from 'classnames';
+
+import {
+  fetchAllMatches,
+  fetchMatchesFromContent,
+  fetchMatchesFromSignal,
+} from '../Api';
+import {timeAgo} from '../utils/DateTimeUtils';
+import ContentMatchPane from '../components/ContentMatchPane';
+import {useQuery} from '../utils/QueryParams';
+
+export default function MatchIndex() {
+  const query = useQuery();
+  const searchQuery = query.get('contentId') || query.get('signalId');
+  let searchInAttribute;
+
+  ['contentId', 'signalId'].forEach(attribute => {
+    if (query.has(attribute)) {
+      searchInAttribute = attribute;
+    }
+  });
+
+  const [
+    selectedContentAndSignalIds,
+    setSelectedContentAndSignalIds,
+  ] = useState({
+    contentId: undefined,
+    signalId: undefined,
+    signalSource: undefined,
+  });
+
+  return (
+    <div className="d-flex flex-column justify-content-start align-items-stretch h-100 w-100">
+      <div className="flex-grow-0">
+        <Container className="bg-dark text-light" fluid>
```

Comment:
re: comment in App.jsx, I think this Container is what now gives the page the padding and why it is missing from the others. I think we should try and have consistent header.
(also super nit: but the dark gray or bg-dark clashes with the gray of selected tab in the sidebar)

Reply:
Yeah. I explained the reasoning for the padding as a reply to your comment above.

re: the dark gray. Totally up for grabs. We can try a few classes and see what works best. Leaving as-is for now because it is a non-functional aspect.